app/controllers/admin/portfolios_controller.rb

```diff
@@ -0,0 +1,52 @@
+class Admin::PortfoliosController < AdminsController
+  def add_portfolio
+    portfolio = Portfolio.create(:name        => params[:name],
+                                 :description => params[:description],
+                                 :image_url   => params[:url],
+                                 :enabled     => params[:enabled])
```

Comment:
We gotta start using strong params in these controllers to secure up the parameter access.

Reply:
@Fryguy - Yeah, was going to look up where to turn that on.

Reply:
It's more of a pattern.  params is already a strong params hash, and instead of accessing it directly, you access it through a helper method that call `.require` on the params to whitelist specific values.

See https://edgeapi.rubyonrails.org/classes/ActionController/StrongParameters.html

Reply:
@Fryguy - Do you have any thoughts on `admin/` namespacing for the controllers?  I'm used to the pattern I implemented, but without an understanding of how Insights will handle 'admin' requests I just went with what I know.